/**:
  ros__parameters:
    # Takeoff behavior
    takeoff_height: 1.0
    takeoff_speed: 0.5
    takeoff_threshold: 0.3
    plugin_name: "takeoff_plugin_speed"
    
    # Land behavior  
    land_speed: 0.2
    land_plugin_name: "land_plugin_speed"
    
    # Go to behavior
    go_to_speed: 0.5
    go_to_threshold: 0.3
    go_to_plugin_name: "go_to_plugin_position"
    
    # Follow path behavior
    follow_path_speed: 0.5
    follow_path_threshold: 0.3
    follow_path_plugin_name: "follow_path_plugin_position"